app/src/main/java/com/h/pixeldroid/api/PixelfedAPI.kt

```diff
@@ -162,5 +162,37 @@ interface PixelfedAPI {
                 .build().create(PixelfedAPI::class.java)
         }
     }
+
+    @Multipart
+    @POST("/api/v1/media")
+    fun mediaUpload(
+        //The authorization header needs to be of the form "Bearer <token>"
+        @Header("Authorization") authorization: String,
+        @Part file: MultipartBody.Part
+    ): Call<Attachment>
+
+    //Used in our case to post a comment or a new post
+    @FormUrlEncoded
+    @POST("/api/v1/statuses")
```

Comment:
This method was already implemented, try to use the existing implementation rather than recreating a second one.